packages/toolkit/package.json

```diff
@@ -59,7 +59,7 @@
         "buildScripts": "npm run generateNonCodeFiles && npm run copyFiles",
         "generateNonCodeFiles": "ts-node ./scripts/build/generateNonCodeFiles.ts",
         "copyFiles": "ts-node ./scripts/build/copyFiles.ts",
-        "clean": "ts-node ../../scripts/clean.ts dist/ LICENSE NOTICE quickStart* CHANGELOG.md",
```

Comment:
@hayemaxi do we want to remove `CHANGELOG.md` in the upstream `master` or only in `feature/standalone`? I.e should we keep `CHANGELOG.md` here, but remove it in `feature/standalone`

Reply:
yeah, in `master` there is no individual extension changelog support, so its maintained at the top-level and copied down to the extension for packaging. So we should keep cleaning it up in in this branch. In `feature/standalone`, there is no top-level changelog; all the changelogs are the true changelog/not copies and shouldn't be cleaned.

This will probably continue to be a problem for us in merges, maybe we can cherry-pick the changelog commits and give `master` the same functionality.

Reply:
ack. Will handle the merge conflicts if they are happening for now.